lighthouse-core/test/fraggle-rock/scenarios/api-test-pptr.js

```diff
@@ -90,10 +89,9 @@ describe('Fraggle Rock API', () => {
         failedAudits,
         notApplicableAudits,
       } = getAuditsBreakdown(lhr);
-      // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`46`);
+      expect(auditResults.map(audit => audit.id)).toMatchSnapshot();
 
-      expect(notApplicableAudits.length).toMatchInlineSnapshot(`5`);
+      expect(notApplicableAudits.map(audit => audit.id)).toMatchSnapshot();
```

Comment:
at least some of these notApplicable don't seem to have anything to do with FR-ness, just that they were notApplicable for this run. Seems like this will be covered by other tests now

Reply:
I think the reasoning behind this is that the N/A audits would vary a lot depending on if any network requests were fetched during the timespan.